<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gameWindow</class>
 <widget class="QWidget" name="gameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gameWindow</string>
  </property>
  <widget class="QPushButton" name="backToMain">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startBut">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>60</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开始游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetBut">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>100</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>重置棋盘</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>150</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪黑神话</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前回合：</string>
   </property>
  </widget>
  <widget class="QLabel" name="whoTurn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>210</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪黑神话</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>黑棋</string>
   </property>
  </widget>
  <widget class="QLabel" name="whoWin">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>600</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪黑神话</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>你能打败流萤吗</string>
   </property>
  </widget>
  <widget class="ChessBoard" name="chessBoardWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>571</width>
     <height>571</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>270</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪黑神话</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>倒计时</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>410</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪黑神话</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>倒计时</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="whiteTimer">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>470</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <property name="digitCount">
    <number>3</number>
   </property>
   <property name="mode">
    <enum>QLCDNumber::Mode::Dec</enum>
   </property>
   <property name="value" stdset="0">
    <double>300.000000000000000</double>
   </property>
   <property name="intValue" stdset="0">
    <number>300</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="blackTimer">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>340</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <property name="digitCount">
    <number>3</number>
   </property>
   <property name="mode">
    <enum>QLCDNumber::Mode::Dec</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::SegmentStyle::Filled</enum>
   </property>
   <property name="value" stdset="0">
    <double>300.000000000000000</double>
   </property>
   <property name="intValue" stdset="0">
    <number>300</number>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChessBoard</class>
   <extends>QWidget</extends>
   <header>chessboard.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
